.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
should we add 'bug' or a default importance level?

Reply:
I will add the priority once I review the issue we have P1, P2, P3 & P4 labels created in this repo.